Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM python:3.7-alpine as base
+RUN pip install pipenv
+RUN mkdir -p /usr/src/app/nhais
+# TODO: should just be . after NIAD-193
```

Comment:
No NIAD ticket against this PR, but is it 193? If we're using the the common repo these need updating

Reply:
Make sure to go through the readme and test creating and running all of the docker containers and docker-compose. They should all still work after the move.